src/lib.rs

```diff
@@ -412,6 +418,15 @@ to advance the applied index inside.
 
 For more information, check out an [example](examples/single_mem_node/main.rs#L113-L179).
 
+Sometimes it's better not to block the raft machine in IO operation, so that latency of
+read/write can be more predictable and the fsync frequencey can be controlled. The crate
+supports async ready to offload the IO operation to other thread. The usage is the same as
+above except:
+1. all writes are not required to be persisted, they can be written into memory caches;
+2. persisted messages should be sent after all coresponding writes are persisted;
+3. [`advance_append_async`](RawNode::advance_append_async) is used when all writes are finished
+    instead of `advance/advance_append`.
```

Comment:
Should also mention what function to call when the writes are persisted.

Reply:
It's mentioned in `advance_append_async`.